rustls/tests/api.rs

```diff
@@ -4753,6 +4753,7 @@ fn tls13_stateful_resumption() {
     // full handshake
     let (mut client, mut server) = make_pair_for_arc_configs(&client_config, &server_config);
     let (full_c2s, full_s2c) = do_handshake(&mut client, &mut server);
+    assert_eq!(client.tls13_tickets_received(), 2);
```

Comment:
If the unbuffered conn API is updated it'd be nice to sneak an update in to one of its unit tests in `rustls/tests/unbuffered.rs`. Maybe the `eary_data()` test?

Reply:
I added the assert.